server/app.js

```diff
@@ -44,3 +49,72 @@ const server = app.listen(3000, () => {
     const port = server.address().port
     console.log('Server is listening on port', port)
 })
+
+
+// // require and instantiate express
```

Comment:
Can you remove all of this commented out code? It's not doing anything for your server, so it's very distracting and makes the PR look bigger than it actually is